### ISSUE TITLE
How do I find where to put the settings?

### ISSUE THREAD

**Issue by JimNayzium** [aaron-bond/better-comments#326]
I put the settings from the default 5 into my settings.json and altered them and nothing is working. I do get the default ones to work when I type the symbols listed but I don't know where or how to add these to a file that I can update to change and customize it?

**Comment by JimNayzium** [comment]
I think I got it -- it was odd but I had to close VSCODE and open it again for anything to work. I used to the settings just taking effect immediately with the other things I have installed. I finally just restarted VSCODE and it worked. Sort of a bummer to have to restart on any formatting change to the settings.json file but it is still a great extension!

**Comment by aaron-bond** [comment]
Yep, this is a limitation of the settings loading logic. It happens on first load and there's no listener for settings being changed to update. I'll keep this bug open and keep a fix in my to-do for this
